include/openthread/history_tracker.h

```diff
@@ -516,6 +528,25 @@ const otHistoryTrackerFavoredOmrPrefix *otHistoryTrackerIterateFavoredOmrPrefixH
     otHistoryTrackerIterator *aIterator,
     uint32_t                 *aEntryAge);
 
+/**
+ * Iterates over the entries in the favored on-link prefix history list.
+ *
+ * Requires `OPENTHREAD_CONFIG_BORDER_ROUTING_ENABLE` (device acting as Border Router).
+ *
+ * @param[in]     aInstance  A pointer to the OpenThread instance.
+ * @param[in,out] aIterator  A pointer to an iterator. MUST be initialized or the behavior is undefined.
+ * @param[out]    aEntryAge  A pointer to a variable to output the entry's age. MUST NOT be NULL.
+ *                           Age is provided as the duration (in milliseconds) from when entry was recorded to
+ *                           @p aIterator initialization time. It is set to `OT_HISTORY_TRACKER_MAX_AGE` for entries
+ *                           older than max age.
```

Comment:
```suggestion
 *                           Age is provided as the duration (in milliseconds) from when the entry was recorded to
 *                           @p aIterator at initialization time. It is set to `OT_HISTORY_TRACKER_MAX_AGE` for entries
 *                           older than the max age.
```

Reply:
Thanks, @Vyrastas.

I've updated everything else in the new pushed commit, but I did not add the suggested `at` above.

To ensure age is reported correctly and consistently, we calculate it relative to the time the iterator is initialized. This provides a stable reference point, ensuring that even if an iteration function like `otHistoryTrackerIterate...()` is called multiple times, we get a consistent age for all entries retrieved with the same iterator object.

Reply:
Thanks for the explanation, I misread the text